src/api/routes/samples.js

```diff
@@ -2,31 +2,50 @@ const SamplesService = require('../route-services/samples');
 
 const samplesService = new SamplesService();
 
+const {
+  expressAuthorizationMiddleware,
```

Comment:
Added auth for the samples endpoints because it didn't have before

Reply:
There's also other endpoints without auth that we shoudl look into, let me know if you want me to work on that in this PR

Reply:
Hmm.. I think it's better to create a new ticket. Don't want to drag this ticket too long.

Reply:
it does look weird though. Why don't we have the auth middleware for samples.

Reply:
Ok, then we need to add a ticket for this